src/marketplaceFonts.css

```diff
@@ -104,7 +104,7 @@
     font-weight: var(--fontWeightBold);
     font-size: 30px;
     line-height: 36px;
-    letter-spacing: -0.5px;
+    letter-spacing: -1px;
```

Comment:
This should be mentioned in the Changelog.
If one of our customer has changed the font (or is just happy with current *letter-spacing*), it might need some adjustment from their own code base.
`- marketplaceH1FontStyles: changed letter spacing to be more tight.`